src/kata5.reachDestination.js

```diff
@@ -1,3 +1,6 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+  const arrivalTime = Math.ceil((distance / speed) * 2) / 2;
+  return `I will be there in ${arrivalTime}, see you soon.`;
```

Comment:
Be there in how long? hours? minutes? years? You should include these units in the string. e.g.
```
 return `I will be there in ${arrivalTime} hours, see you soon.`;
```
However, then if `arrivalTime` is `1` your sentence would then be grammatically incorrect `"I will be there in 1 hours, see you soon."` instead of `"I will be there in 1 hour, see you soon."`. To fix this, you can conditionally add the `"s"` at the end of `"hour"` e.g.
```
return `I will be there in ${arrivalTime} hour${arrivalTime === 1 ? "" : "s"}, see you soon.`;
```